packages/platforms/react-native/__mocks__/react-native.ts

```diff
@@ -101,6 +101,14 @@ type DeviceInfo
   | DeviceInfoAndroid
   | Record<string, never>
 
+const createPool = () => {
+  const pool: number[] = []
+  for (let i = 0; i < pool.length; i++) {
```

Comment:
`pool.length` here is always 0 so this loop is never entered — should this function just be `return []` or should it be filling the array?

Reply:
yep, that's a bug!

This has changed now anyway - after some discussion we decided to refactor the native implementation to return the random bytes as a 2048 character hex encoded string